jumpscale/sals/kubernetes/manager.py

```diff
@@ -97,13 +97,24 @@ def add_helm_repo(self, name, url):
         return out
 
     @helm_required
-    def install_chart(self, release, chart_name, namespace="default", extra_config=None, chart_values_file=None):
-        """deployes a helm chart
+    def install_chart(
+        self,
+        release,
+        chart_name,
+        namespace="default",
+        extra_config=None,
+        chart_values_file=None,
+        extra_config_string_safe=None,
+        timeout="7m0s",
```

Comment:
is this timeout format valid?

Reply:
yes it is valid from helm official docs 
https://helm.sh/docs/intro/using_helm/#helpful-options-for-installupgraderollback
![image](https://user-images.githubusercontent.com/46320202/116088906-9afb3800-a6a2-11eb-8091-9b3d1c0a3d89.png)